mysql-test/suite/sys_vars/t/transaction_read_only_basic.test

```diff
@@ -1,6 +1,6 @@
-############## mysql-test\t\tx_read_only_basic.test ###########################
+############## mysql-test\t\transaction_read_only_basic.test ##################
```

Comment:
Update here the path.
Apply general comment from `mysql-test/suite/sys_vars/t/transaction_isolation_basic.test`